plugins/oauth/web_client/stylesheets/login.styl

```diff
@@ -29,15 +31,33 @@
     border-right 1px solid white
   .g-oauth-button-text
     display inline-block
-    padding 0 20px
+    text-align center
+    width 100px
```

Comment:
The max amount we have to expand into is 109px (110 causes it to flow down below the icon). This is what it looks like as 100:

<img width="458" alt="screen shot 2015-11-12 at 11 20 56 am" src="https://cloud.githubusercontent.com/assets/555959/11123742/7db0b8b0-892f-11e5-91d5-ce1495fda1a2.png">

If you'd rather go to the full 109px right now, we can do that; if we want to go any further we'll need to bump up the button width over 160px.